composinator/server.go

```diff
@@ -0,0 +1,61 @@
+package composinator
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/gorilla/mux"
+	v3 "github.com/rancher/go-rancher/v3"
+)
+
+type convertOptions struct {
+	StackID string `json:"stackId,omitempty" yaml:"stackId,omitempty"`
+	Format  string `json:"format,omitempty" yaml:"format,omitempty"`
+}
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	if r.Body != nil {
+		defer r.Body.Close()
```

Comment:
on the server-side you don't have to worry about closing http input/output.  The server will do that after the handler method is called